src/ResourceManager/Profile/Commands.Profile/Commands.Profile.Netcore.csproj

```diff
@@ -36,9 +36,17 @@
   <ItemGroup>
     <PreLoadAssemblies Include="$(ProjectDir)..\..\..\lib\Newtonsoft.Json.9.dll"/>
     <PreLoadAssemblies Include="$(ProjectDir)..\..\..\lib\Newtonsoft.Json.10.dll"/>
+    <PreLoadAssemblies Include="$(ProjectDir)..\..\..\lib\Microsoft.IdentityModel.Clients.ActiveDirectory.dll"/>
```

Comment:
@markcowl discussed offline about changing the path to be

`lib\Microsoft.IdentityModel.Clients.ActiveDirectory\{{version}}\Microsoft.IdentityModel.Clients.ActiveDirectory.dll`